app/modules/assignment/actions/assignment-fetch-info.js

```diff
@@ -0,0 +1,29 @@
+import {receiveInfo} from "./assignment-receive-info"
+import {requestInfo} from "./assignment-request-info"
+import {errorInfo} from "./assignment-error-info"
+
+import {url} from "../selectors"
+
+export const assignmentFetchInfo = () => {
+  return (dispatch, getState) => {
+    try {
+      var urlObj = new URL(url(getState()))
+      var infoURL = `${urlObj.origin}/api/internal/${urlObj.pathname}/info`
+    } catch (e) {
+      dispatch(errorInfo("URL is invalid!"))
+      return
+    }
+    dispatch(requestInfo())
+    return window.fetch(infoURL, {
+      credentials: "include"
+    })
+      .then(response => response.json())
+      .then((data) => {
+        dispatch(receiveInfo(data.name, data.type))
+      })
+      .catch((e) => {
+        console.log(e)
```

Comment:
:art: is this still necessary for developing purposes? I'm fine with leaving it around if it is, these are easy to detect with [`no-console`](https://eslint.org/docs/rules/no-console) in `eslint` at a later stage.

Reply:
Removed all console logs, they are useful for debugging for right now but I can just add them back when developing.